ddpui/tests/utils/test_helpers.py

```
@@ -232,3 +234,15 @@ def test_get_integer_env_var_5():
     with patch("os.getenv") as mock_getenv:
         mock_getenv.return_value = "-20"
         assert get_integer_env_var("TEST_VAR", 21, None, True) == -20
+
+
+def test_extract_year_month_valid():
+    assert extract_year_month("2025-07-21")=="2025-07"
+    assert extract_year_month("1999-01-01")=="1999-01"
+    
+import pytest
+from ddpui.utils import extract_year_month  # or adjust import as needed
```

Ruff (0.12.2)

Redefinition of unused `extract_year_month` from line 13

(F811)